### PR TITLE
use lucode2::showUpdates, Tutorial: exclude renv from update

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
# Purpose of this PR
To remove code duplication lucode2::showUpdates was added. The renv tutorial showed `renv::update()`, added `exclude = "renv"` there.

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Refactoring

## Checklist:

- [x] My code follows the coding etiquette
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: scripts/start/submit.R [hunk 1]
```diff
@@ -62,26 +62,15 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
...
-          outdatedPackages <- utils::old.packages(instPkgs = installed[installed[, "Package"] %in% pikPackages, ])
-          if (!is.null(outdatedPackages)) {
-            message("The following PIK packages can be updated:\n",
-                    paste("-", outdatedPackages[, "Package"], ":",
-                          outdatedPackages[, "Installed"], "->", outdatedPackages[, "ReposVer"],
-                          collapse = "\n"),
-                    "\nConsider updating with `Rscript scripts/utils/updateRenv.R`.")
-          }
+        } else if (!is.null(lucode2::showUpdates())) {
+          message("Consider updating with `Rscript scripts/utils/updateRenv.R`.")
```
Comment (pfuehrlich-pik) [inline scripts/start/submit.R:66]: Should we tell users to use make now in these cases @mikapfl ?
```suggestion
          message("Consider updating with `make update-renv`.")
```

**Review by orichters — APPROVED** [review]

File: scripts/start/submit.R (hunk 1 above)
Comment (mikapfl) [reply scripts/start/submit.R:66]: For linux users, `make` will be easier. People running on Windows probably want `Rscript`. I'd say, leave it as it is.

**Review by mikapfl — CHANGES_REQUESTED** [review]
The lucode2 version needs to be updated also here: https://github.com/remindmodel/remind/blob/64f0d719855e612cb7118028b18924d895dbc4af/start.R#L216

That's not nice, but no way around, since we use lucode2 to check the versions in the DESCRIPTION so there is a circular dependency if just adding it to the DESCRIPTION.

**Comment by pfuehrlich-pik** [comment]
I don't think so, but maybe I'm missing something. We need lucode2 >= 0.34.0 so we have access to `lucode2::checkDeps`. Once we have that all further required lucode2 updates are enforced/suggested by checkDeps, no?

**Review by mikapfl — APPROVED** [review]

**Pull request merged**